x/supplier/client/cli/query_claim_test.go

```diff
@@ -54,15 +57,47 @@ func TestClaim_Show(t *testing.T) {
 			supplierAddr: claims[0].GetSupplierAddress(),
 
 			args: common,
-			err:  status.Error(codes.NotFound, "not found"),
+
+			expectedErr: status.Error(
+				codes.NotFound,
+				types.ErrSupplierClaimNotFound.Wrapf(
+					// TODO_CONSIDERATION: factor out error message format strings to constants.
+					"session ID %q and supplier %q",
+					"wrong_session_id",
+					claims[0].GetSupplierAddress(),
+				).Error(),
+			),
+		},
+		{
+			desc:         "claim not found (invalid bech32 supplier address)",
+			sessionId:    claims[0].GetSessionHeader().GetSessionId(),
+			supplierAddr: "invalid_bech32_supplier_address",
+
+			args: common,
+			// NB: this is *NOT* a gRPC status error because the bech32 parse
+			// error occurs during request validation (i.e. client-side).
+			expectedErr: types.ErrSupplierInvalidAddress.Wrapf(
+				// TODO_CONSIDERATION: prefer using "%q" in error format strings
```

Comment:
I think this is an awesome idea. Let's start doing this!

Might just be worth @mentioning protocol-dev on discord and we'll adopt it over time.